packages/contracts-bedrock/scripts/deploy/UpgradeAnchorStateRegistry.s.sol

```diff
@@ -136,7 +135,7 @@ contract UpgradeAnchorStateRegistry is Script {
             })
         });
         return
-            abi.encodeWithSelector(
+            abi.encodeCall(
                 IAnchorStateRegistry.initialize.selector,
```

Comment:
Should be `IAnchorStateRegistry.initialize` if `abi.encodeCall` is used instead.